libraries/AP_Follow/AP_Follow.h

```diff
@@ -153,33 +186,71 @@ class AP_Follow
     // write out an onboard-log message to help diagnose follow problems:
     void Log_Write_FOLL();
 
-    // parameters
-    AP_Int8     _enabled;           // 1 if this subsystem is enabled
-    AP_Int16    _sysid;             // target's mavlink system id (0 to use first sysid seen)
-    AP_Float    _dist_max_m;          // maximum distance to target.  targets further than this will be ignored
-    AP_Int8     _offset_type;       // offset frame type (0:North-East-Down, 1:RelativeToLeadVehicleHeading)
-    AP_Vector3f _offset_ned_m;            // offset from lead vehicle in meters
-    AP_Int8     _yaw_behave;        // following vehicle's yaw/heading behaviour (see YAW_BEHAVE enum)
-    AP_Int8     _alt_type;          // altitude source for follow mode
-    AC_P        _p_pos;             // position error P controller
-    AP_Int16    _options;           // options for mount behaviour follow mode
-
-    // local variables
-    uint32_t _last_location_update_ms;  // system time of last position update
-    Vector3p _target_position_ned_m;  // last known position of target
-    Vector3f _target_velocity_ned_ms;  // last known velocity of target in NED frame in m/s
-    Vector3f _target_accel_ned_mss;     // last known acceleration of target in NED frame in m/s/s
-    uint32_t _last_heading_update_ms;   // system time of last heading update
-    float _target_heading_deg;          // heading in degrees
-    bool _automatic_sysid;          // did we lock onto a sysid automatically?
-    float _dist_to_target_m;          // latest distance to target in meters (for reporting purposes)
-    float _bearing_to_target_deg;       // latest bearing to target in degrees (for reporting purposes)
-    bool _offsets_were_zero;        // true if offsets were originally zero and then initialised to the offset from lead vehicle
-
-    // setup jitter correction with max transport lag of 3s
-    JitterCorrection _jitter{3000};
+    //==========================================================================
+    // Parameters
+    //==========================================================================
+
+    AP_Int8     _enabled;
+    AP_Int16    _sysid;
+    AP_Float    _dist_max_m;
+    AP_Int8     _offset_type;
+    AP_Vector3f _offset_XXD_m;
+    AP_Int8     _yaw_behave;
+    AP_Int8     _alt_type;
+    AC_P        _p_pos;
+    AP_Int16    _options;
+    AP_Float    _accel_max_ne_mss;
```

Comment:
overall this looks pretty good but I wonder why we've removed all the comments to the right of the variables and parameters?